docs/source-pytorch/extensions/strategy.rst

```diff
@@ -81,12 +81,6 @@ The below table lists all relevant strategies available in Lightning with their
    * - deepspeed
      - :class:`~lightning.pytorch.strategies.DeepSpeedStrategy`
      - Provides capabilities to run training using the DeepSpeed library, with training optimizations for large billion parameter models. :ref:`Learn more. <advanced/model_parallel:deepspeed>`
-   * - hpu_parallel
```

Comment:
I would keep these, with a note that they live in a separate package

Reply:
resolved